pkg/testing/testdata/testdata.go

```diff
@@ -419,6 +466,55 @@ const (
                     }
                 ]
             }
+        },
+		{
+            "apiVersion": "route.openshift.io/v1",
+            "kind": "Route",
+            "metadata": {
+                "annotations": {
+                    "openshift.io/host.generated": "true"
+                },
+                "creationTimestamp": "2018-04-03T10:52:09Z",
+                "labels": {
+                    "app": "ruby-ex"
+                },
+                "name": "ruby-ex-tls",
+                "namespace": "myproject",
+                "resourceVersion": "3217",
+                "selfLink": "/apis/route.openshift.io/v1/namespaces/myproject/routes/ruby-ex-tls",
+                "uid": "0eeabedc-372d-11e8-853f-567fb20565f6"
+            },
+            "spec": {
+                "host": "ruby-ex-tls-myproject.192.168.42.14.nip.io",
+                "port": {
+                    "targetPort": "8080-tcp"
+                },
+                "tls": {
```

Comment:
this is tested? if so... where can I see the expected outcome?

Reply:
yes, this is tested.. the outcome would be we have the route starting with https instead of http.. i added a  service and route spec in `pkg/testing/testdata.go`